src/main/java/org/broadinstitute/hellbender/tools/exome/hmm/CopyNumberTriStateHiddenMarkovModel.java

```diff
@@ -16,36 +15,18 @@
 import java.util.*;
 
 /**
- * Implements the Three state copy number distance dependent model described in
- * <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3484655">Fromer et al. Am J Hum Genet. 2012 Oct 5; 91(4):597-607</a>
- *
- * <p>
- *     This model has the following parameters:
- *     <dl>
- *         <dt>Event-Rate</dt><dd>(p in the paper) the exome-wide event rate; what is the probability of any given
- *         target to be the start target for an event (either a deletion or duplication).</dd>
- *         <dt>Mean number of targets per event</dt><dd>(T in the paper) the average number of targets per each event
- *         (either a deletion or duplication).</dd>
- *         <dt>Mean event target distance</dt><dd>(meanEventSize in the paper) the expectation of the distance (in base-pairs) between
- *         consecutive targets in an event.</dd>
- *         <dt>Mean coverage depth shift for deletions</dt><dd>(-M in the paper) what is the average negative shift
- *         in coverage in regions that have undergone a copy loss.</dd>
- *         <dt>Mean coverage depth shift for duplications</dt><dd>(+M in the paper) what is the average positive
- *         shift in coverage depth in regions that have undergone a copy gain</dd>
- *     </dl>
- *     <p>
- *      Notice that this implementation allows a different shift magnitude for deletions and duplications which is fix
- *      to the same value in the original paper.
- *     </p>
- * </p>
+ *  This model has the following parameters:
```

Comment:
Since we have never used javadoc formatting, I chose to make it more human-readable in the code.